script/nan-spec-runner.js

```diff
@@ -25,7 +25,43 @@ async function main () {
     npm_config_arch: process.env.NPM_CONFIG_ARCH,
     npm_config_yes: 'true'
   });
-  const { status: buildStatus } = cp.spawnSync(NPX_CMD, ['node-gyp', 'rebuild', '--directory', 'test', '-j', 'max'], {
+
+  const clangDir = path.resolve(BASE, 'third_party', 'llvm-build', 'Release+Asserts', 'bin');
+  const cc = path.resolve(clangDir, 'clang');
+  const cxx = path.resolve(clangDir, 'clang++');
+  const ld = path.resolve(clangDir, 'lld');
+
+  // TODO(ckerr) this is cribbed from read obj/electron/electron_app.ninja.
+  // Maybe it would be better to have this script literally open up that
+  // file and pull cflags_cc from it instead of using bespoke code here?
+  // I think it's unlikely to work; but if it does, it would be more futureproof
```

Comment:
IMO no, for the following reasons:

1. electron_app.ninja has lots of unrelated flags (e.g. include paths) which could potentially adversely affect this compilation.
2. It's useful to know what the minimal set of flags needed is, especially since we will need to mirror those flags over to electron-rebuild.

Reply:
I'll remove this comment entirely so we don't go down the wrong path in the future